src/api/common/utils.js

```diff
@@ -20,8 +20,20 @@ function convertAmount(amount) {
   };
 }
 
+function wrapCatch(asyncFunction: any): any {
```

Comment:
> Too Few Arguments
Flow produces an error whenever a call provides too few arguments to a function. On the other hand, too many arguments is fine, which is why Flow does not complain on most uses of builtin primitives like Array, whose methods are often called with closures that take fewer parameters than prescribed.


http://flowtype.org/docs/functions.html#too-few-arguments